package.json

```diff
@@ -9,11 +9,12 @@
         "production": "cross-env NODE_ENV=production webpack",
         "build": "npm run production",
         "watch": "cross-env webpack --watch",
+        "installcodestandards": "composer run-script install-codestandards",
```

Comment:
@bradp what is use of this command?

Reply:
Will let you manually run the composer script to install the code standards into the /vendor folder.

Reply:
@bradp Can you tell me the use case where the developer has to run this manually?

Reply:
Anyone who has a copy of the repo can easily run it to install the scripts without having to reset their local install. Mostly it's there to be a helper.

Reply:
@JasonTheAdams Do you have any thoughts on this?

Reply:
I don't personally see any value in running `npm run installcodestandards` versus `composer run-script install-codestandards`. This is really just a pre-defined npm script to run a composer script.